etc/ci/manifest_changed.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
-./mach test-wpt --manifest-update SKIP_TESTS > /dev/null
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# We shouldn't need any binary at all to update the manifests.
+# Adding "SKIP_TESTS" to skip tests, and "--binary=" to skip binary check.
```

Comment:
Can you expand more about how SKIP_TESTS works in this comment? The name makes it clear that it's used to skip the tests, but we should document how it does so.